pkg/idtools/idtools_unix.go

```diff
@@ -1,20 +1,76 @@
 //go:build !windows
 
-package idtools // import "github.com/docker/docker/pkg/idtools"
+package idtools
 
 import (
-	"bytes"
 	"fmt"
-	"io"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"strconv"
 	"syscall"
 
 	"github.com/moby/sys/user"
 )
 
+// LoadIdentityMapping takes a requested username and
+// using the data from /etc/sub{uid,gid} ranges, creates the
+// proper uid and gid remapping ranges for that user/group pair
+func LoadIdentityMapping(name string) (IdentityMapping, error) {
```

Comment:
Could you keep these at the same location they were in to reduce code-churn / make it easier to see the differences?